stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AccessibilityNodeInfoWrapper.java

```diff
@@ -37,33 +38,66 @@ public static boolean getIgnored(View view, AccessibilityNodeInfoCompat nodeInfo
       parent = parent.getParent();
     }
 
-    // AccessibilityNodeInfo is actionable
-    if (nodeInfo.isClickable() || nodeInfo.isLongClickable() || nodeInfo.isFocusable()) {
-      return false;
+    if (!node.isVisibleToUser()) {
+      return true;
     }
 
-    // AccessibilityNodeInfo has text set (commonly set from TextViews and Switches).
-    CharSequence nodeText = nodeInfo.getText();
-    if (nodeText != null && nodeText.length() > 0) {
-      return false;
+    if (AccessibilityUtil.isAccessibilityFocusable(node, view)) {
+      if (node.getChildCount() <= 0) {
+        // Leaves that are accessibility focusable are never ignored, even if they don't have a
+        // speakable description
+        return false;
+      } else if (AccessibilityUtil.isSpeakingNode(node, view)) {
+        // Node is focusable and has something to speak
+        return false;
+      }
+
+      // Node is focusable and has nothing to speak
+      return true;
     }
 
-    // AccessibilityNodeInfo has a content description
-    CharSequence contentDescription = nodeInfo.getContentDescription();
-    if (contentDescription != null && contentDescription.length() > 0) {
+    // If this node has no focusable ancestors, but it still has text,
+    // then it should receive focus from navigation and be read aloud.
+    if (!AccessibilityUtil.hasFocusableAncestor(node, view) && AccessibilityUtil.hasText(node)) {
       return false;
     }
 
-    // View has an AccessibilityNodeProvider
-    if (ViewCompat.getAccessibilityNodeProvider(view) != null) {
-      return false;
+    return true;
+  }
+
+  public static String getIgnoredReasons(AccessibilityNodeInfoCompat node, View view) {
+    int important = ViewCompat.getImportantForAccessibility(view);
+
+    if (important == ViewCompat.IMPORTANT_FOR_ACCESSIBILITY_NO) {
+      return "View has importantForAccessibility set to 'NO'.";
```

Comment:
Yeah, since most of the CSS property names will never be localized since they are derived from the method names on the view, I figure it's probably okay to also have the values not be localized.  This would be easy enough to switch out later if we want though.